edgelet/README.md

```diff
@@ -2,6 +2,9 @@
 This is the code repository for the IoT Edge Security Daemon.
 The daemon provides an abstraction over Hardware Security Modules (HSMs), enabling secure SAS token and certificate provisioning for IoT Edge modules.
 
+## Test IoT Edge daemon API endpoints on dev machine
```

Comment:
Can we move this link and description to the `devguide`? I think we should keep this readme clean and simple, and put the more interesting info in the dev guide.